File: tests/fixtures/pqc_sample.go
// Fixture for PQC-ready detection: Open Quantum Safe liboqs-go (Kyber, Dilithium)
package main

import (
	"github.com/open-quantum-safe/liboqs-go/oqs"
)

func main() {
	kem := oqs.KeyEncapsulation{}
	defer kem.Free()
	_ = kem.Init("Kyber512", nil)
}

File: tests/fixtures/pqc_sample.py
# Fixture for PQC-ready detection: Open Quantum Safe (pyoqs) – Kyber, Dilithium
import oqs

def kem_example():
    kem = oqs.KeyEncapsulation("Kyber512")
    public_key = kem.generate_keypair()
    ciphertext, shared_secret = kem.encap_secret(public_key)
    return ciphertext, shared_secret
